app/javascript/src/utils/text.js

```diff
@@ -0,0 +1,3 @@
+export function toCapitalize(string) {
+  return string.toLowerCase().replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase())
```

Comment:
While we are here...

```suggestion
  return string.toLowerCase().replace(/(?<=^|\s)\w/g, letter => letter.toUpperCase())
```

This takes less steps 
![previous regex](https://github.com/Mitcheljager/workshop.codes/assets/6181929/3623962f-1f05-4e26-9b80-7ae1d0729d8e) (https://regex101.com/r/Mlbicn/1)
![new regex](https://github.com/Mitcheljager/workshop.codes/assets/6181929/05a081a7-67a8-4266-bd40-373499741c07) (https://regex101.com/r/Mlbicn/2)

Reply:
Hmm, fair enough, but I will keep it out of this PR

Reply:
I didn't realize I had the flavor set to PHP. Here are some results with JS:

![previous regex](https://github.com/Mitcheljager/workshop.codes/assets/6181929/b469c3b7-24ea-43eb-a92a-c302436029cd) (https://regex101.com/r/Mlbicn/3)
![new regex](https://github.com/Mitcheljager/workshop.codes/assets/6181929/41806c92-9e20-47c4-bcd9-10d77e0eafa6) (https://regex101.com/r/Mlbicn/4)

Doesn't change anything, but I wanted to keep the record straight :P